### PR TITLE
Dashboard alterations

### LINKED ISSUE #162: Remove error box at subscriptions page/dashboard
When you first enter the subscriptions page, there's an error box that already says missing field. Ideally, this box should only show when someone doesn't enter an email, rather than appearing from the start

### PR DESCRIPTION
**Pull request by e-lemma**
Several changes to the dashboard including:

- Minor formatting improvements
- Resolves #162 by checking for valid inputs only after the submit button as been pressed
- Hiding empty graphs, only displaying them when user selects data to be displayed
- Adding a name and image to the dashboard browser tab

Have a look and let me know if anything should be changed. The changes as described above are now live on the dashboard.

### CODE REVIEW

File: dashboard/pages/artists.py [hunk 1]
```diff
@@ -27,18 +27,21 @@ def show_artists():
         '1 day', '1 week', '1 month', "1 year"], horizontal=True)
 
     pop_artists = get_popular_artists(conn, timeframe)
-    st.header("Top Artists")
```
Comment (Zhi-704) [inline dashboard/pages/artists.py:30]: Slightly hilarious, but I prefer the previous code where there were capitalisation

File: dashboard/pages/home.py [hunk 2]
```diff
@@ -32,14 +32,14 @@ def show_home():
             you can also subscribe to receive email notifications!""")
     timeframe = st.radio(label="Filter by sale timeframe", options=[
         '1 day', '1 week', '1 month', "1 year"], horizontal=True)
-    st.subheader("Top Tracks")
+    st.header("Top tracks")
```
Comment (Zhi-704) [inline dashboard/pages/home.py:35]: Still disagree

**Review by Zhi-704 — APPROVED** [review]
Can't really see anything wrong. Disagree with the capitalisation.

File: dashboard/pages/artists.py (hunk 1 above)
Comment (e-lemma) [reply dashboard/pages/artists.py:30]: Bring that up with the project manager

File: dashboard/pages/home.py (hunk 2 above)
Comment (e-lemma) [reply dashboard/pages/home.py:35]: OK

**Pull request merged**